src/main/resources/application.yml

```diff
@@ -8,8 +8,19 @@ spring:
     show-sql: 'true'
     hibernate:
       ddl-auto: create-drop
-      properties:
```

Comment:
You have changed the count of whitespaces before format_sql -- Does it still work, and why do we need to change that?